project.js

```diff
@@ -0,0 +1,115 @@
+const works = document.querySelector('.work-section');
+const showDetails = document.querySelector('.popup-window');
+const blur = document.getElementById('blur');
+const close = document.querySelector('.close');
+const Ptitle = document.querySelector('.title');
+const Pcanop = document.querySelector('.canop');
+const Pstack1 = document.querySelector('.stack1');
+const Pstack2 = document.querySelector('.stack2');
+const pic = document.querySelector('.detail-pic');
+const desc = document.querySelector('.description');
+const Ptag = document.querySelector('.tag');
+const Plive = document.querySelector('.live');
+const Psource = document.querySelector('.source');
+const popup = [
+  {
+    id: 0,
+    title: 'Tonic',
+    Organization: 'CANOPY',
+    Stack: 'Back End Dev',
+    year: '2015',
+    image: 'images/first.png',
+    Description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```

Comment:
Kindly attempt to make your design similar to the template given.

Your Design | Template
:---: | :---:
<img width="1399" alt="image" src="https://github.com/sheldonwafula/The-portfolio/assets/27709832/83201349-c558-4fbe-973c-d6e35299a91c"> | <img width="671" alt="image" src="https://github.com/sheldonwafula/The-portfolio/assets/27709832/6bfbfd53-825e-4bd8-b56c-e2f306f4d5fb">

- Kindly centre the container on the screen.
- Also ensure the description and the image display side by side preferably using flex/grid.
- The buttons on the popup window should have a hover as shown in the template.